client/src/templates/layouts/Main.js

```diff
@@ -1,22 +1,27 @@
 import styled from "styled-components";
-import { mq } from "constants/theme";
+import { theme, mq } from "constants/theme";
+const { offWhite } = theme.colors;
 
 const Main = styled.main`
   margin-left: 2rem;
   margin-right: 2rem;
 
-  min-height: calc(100vh-124px);
+  display: flex;
+  align-items: center;
+  justify-content: center;
 
-  @media screen and (min-width: ${mq.tablet.wide.minWidth}) {
-    min-height: calc(100vh-166px);
-  }
+  min-height: calc(100% - 119px);
+  min-height: -o-calc(100% - 119px); /* opera */
```

Comment:
We might want to use [autoprefixer](https://www.npmjs.com/package/autoprefixer) in the future but for now I think this is fine